src/rio.h

```diff
@@ -179,5 +180,6 @@ int rioWriteBulkObject(rio *r, struct redisObject *obj);
 
 void rioGenericUpdateChecksum(rio *r, const void *buf, size_t len);
 void rioSetAutoSync(rio *r, off_t bytes);
+void rioEnableReclaimCache(rio *r, int enabled); 
```

Comment:
it can also be used to disable, maybe this is slightly better. WDYT?
```suggestion
void rioSetAutoReclaimCache(rio *r, int enabled); 
```

Reply:
Sure, I'm always not good at naming things... I also expected this function could both turn on and turn off the option.